integration-test/pom.xml

```diff
@@ -51,6 +53,30 @@
 
     <dependency>
       <groupId>${project.groupId}</groupId>
+      <artifactId>livy-repl_2.10</artifactId>
```

Comment:
Why do we need to import repl jars, I think it is not necessary, `InteractiveSession` will pick and upload to Spark automatically if existed in repl/scala-2.xx/target/jars.